tools/wpt/run.py

```diff
@@ -291,11 +291,12 @@ def setup_kwargs(self, kwargs):
             kwargs["package_name"] = "org.mozilla.geckoview.test_runner"
         app = kwargs["package_name"]
 
-        if kwargs["device_serial"] is None:
-            kwargs["device_serial"] = "emulator-5554"
+        if not kwargs["device_serial"]:
+            kwargs["device_serial"] = ["emulator-5554"]
 
         # We're running on an emulator so ensure that's set up
-        if kwargs["device_serial"].startswith("emulator-"):
+        # Assume only one Android device is attached
+        if kwargs["device_serial"][0].startswith("emulator-"):
```

Comment:
Should this start one emulator for each device serial that starts with `emulator-`?

Reply:
I actually do not understand how this works on Firefox, so I made the change here to make sure my change will not break this.

I considered to change this to a for-loop, but the device serial is not used in the lines following that. I would suggest you folks take over this, and make any change that is necessary.

At our side, we have a wrapper around wpt. All the operation to the emulator will be done in the wrapper.

Reply:
updated, pls check again.